src/services/podcast.ts

```diff
@@ -166,10 +173,10 @@ export const insertOrRemovePodcastFromAlphabetizedArray = (podcasts: any[], podc
   } else {
     podcasts.push(podcast)
     podcasts.sort((a, b) => {
-      let titleA = a.title.toLowerCase()
-      let titleB = b.title.toLowerCase()
-      titleA = removeArticles(titleA)
-      titleB = removeArticles(titleB)
+      let titleA = a.sortableTitle.toLowerCase().trim()
+      let titleB = b.sortableTitle.toLowerCase().trim()
+      titleA = titleA.replace(/#/g, '')
+      titleB = titleB.replace(/#/g, '')
       return (titleA < titleB) ? -1 : (titleA > titleB) ? 1 : 0
```

Comment:
holly molly

Reply:
Sorry this was a temporary patch. The sortableTitle's are now being trimmed in the API by the feed parser so our apps won't have to do it.